tools/Azure.Mcp.Tools.Extension/src/Commands/AzCommand.cs

```diff
@@ -138,7 +138,7 @@ private async Task<bool> AuthenticateWithAzureCredentialsAsync(IExternalProcessS
             var azPath = FindAzCliPath() ?? throw new FileNotFoundException("Azure CLI executable not found in PATH or common installation locations. Please ensure Azure CLI is installed.");
 
             var loginCommand = $"login --service-principal -u {credentials.ClientId} -p {credentials.ClientSecret} --tenant {credentials.TenantId}";
-            var result = await processService.ExecuteAsync(azPath, loginCommand, 60, cancellationToken: cancellationToken);
+            var result = await processService.ExecuteAsync(azPath, loginCommand, null, 60, cancellationToken: cancellationToken);
```

Comment:
Named arguments would be helpful here:

```suggestion
            var result = await processService.ExecuteAsync(azPath, loginCommand,
                operationTimeoutSeconds: 60,
                cancellationToken: cancellationToken);
```